### PR TITLE
bat file added⚙️

### PR DESCRIPTION
**Pull request by ashish-shr**

### CODE REVIEW

File: run.bat
Comment (diwacreation3) [inline run.bat:1]: this bat can perform on windows not in linux

**Comment by diwacreation3** [comment]
This push might break template so it's dropped

File: run.bat
Comment (ashish-shr) [reply run.bat:1]: let's keep both then, bat and makefile.

**Review by diwacreation3 — COMMENTED** [review]
Don't remove things about  Makefile 🙂

**Comment by ashish-shr** [comment]
readme updated✨

**Comment by diwacreation3** [comment]
This push has lots of flwas that might break the whole template
So pull request dropped 🥲

**Pull request closed without merging**